src/controllers/rootGroupPermissions.ts

```diff
@@ -0,0 +1,42 @@
+import { Access } from '@prisma/client';
+import { RequestHandler } from 'express';
+import RootGroupPermission from '../models/RootGroupPermission';
+
+export const create: RequestHandler<
+    any,
+    any,
+    { documentRootId: string; studentGroupId: string; access: Access }
+> = async (req, res, next) => {
+    try {
+        const { documentRootId, studentGroupId, access } = req.body;
+
+        if (!(documentRootId && studentGroupId && access)) {
+            res.status(400).send('Missing documentRootId, studentGroupId or access');
```

Comment:
Fail loud and let the error handler handle the rest. Import the HTTP400Error from `../utils/errors/Errors`
```suggestion
            throw new HTTP400Error('Missing documentRootId, studentGroupId or access');
```